client/question/components/LearnerViewDirective.js

```diff
@@ -59,6 +59,10 @@ tie.directive('learnerView', [function() {
                                               on-state-change="scrollToBottomOfFeedbackWindow()">
                         </syntax-error-snippet>
                       </span>
+                      <span ng-if="paragraph.isOutputParagraph()">
+                        <output-snippet content="paragraph.getContent()">
```

Comment:
I don't see the file defining an OutputSnippet directive. Did you forget to "git add"?

(As a general rule, I suggest checking the ["Files Changed" tab](https://github.com/google/tie/pull/438/files) after pushing to GitHub, in order to ensure that the diff is exactly what you want it to be.)

Reply:
Ah... I forgot to git add the new file. I'll add it.

Reply:
I think I addressed this?